JsonParser.h

```diff
@@ -0,0 +1,14 @@
+#include <regex>
+#include <map>
+#include <iostream>
+#include <fstream>
+
+class parser
+{
+
+public:
+
+    std::map <std::string, std::string> jsonParser(std::istream& file);
+    std::map <std::string, std::string> jsonParser(std::string file);
```

Comment:
Mivel nincs adattag, amivel dolgozhatnának, gyanús, hogy lehetnének a függvények statikusak is.